ssz-rs/src/vector.rs

```diff
@@ -1,22 +1,17 @@
 use crate::de::{deserialize_homogeneous_composite, Deserialize, DeserializeError};
+use crate::lib::*;
 use crate::merkleization::{
     merkleize, pack, MerkleCache, MerkleizationError, Merkleized, Node, BYTES_PER_CHUNK,
 };
-use crate::ser::{serialize_composite, Serialize, SerializeError};
-use crate::{SimpleSerialize, SimpleSerializeError, Sized};
+use crate::ser::{serialize_composite, Serialize};
+use crate::{SerializeError, SimpleSerialize, SimpleSerializeError, Sized};
```

Comment:
don't see reason to change import path on `SerializeError`...

Reply:
Agreed, reverted.